climpred/classes.py

```diff
@@ -1121,3 +1122,35 @@ def _verify(
             else:
                 res = _reset_temporal_axis(res, self._temporally_smoothed, dim='lead')
         return res
+
+    def reduce_bias(self, alignment, how='mean', **kwargs):
```

Comment:
I would add an explicit arg here that says `cross_validate=True/False` and drop the `**kwargs` since that's the only keyword.